### ISSUE TITLE
Possible formatter bug with `])` at new line

### ISSUE THREAD

**Issue by SanksTheYokai** [crystal-lang/crystal#7766]
When I do
```crystal
@[Annotation(123
)]
```
And run
`$ crystal tool format --show-backtrace`
I get this error
```
expecting ), not `NEWLINE, `, at :1:17 (Exception)
Error opening file 'crystal' with mode 'r': No such file or directory (Errno)
Unhandled exception: Error opening file 'crystal' with mode 'r': No such file or directory (Errno)
Failed to raise an exception: END_OF_STACK
[0x55b274bd7db6] ???
[0x55b27427907b] __crystal_raise +43
[0x55b27427a865] ???
[0x55b274285e8a] ???
[0x55b27427f2cc] ???
[0x55b27427e80f] ???
[0x55b274289c1c] ???
[0x55b274294d0f] ???
[0x55b27427ce67] main +55
[0x7f614ccdace3] __libc_start_main +243
[0x55b27427551e] _start +46
[0x0] ???
```
But when I change it to
```crystal
@[Annotation(123)]
```
It works fine.

Also:
```
Crystal 0.28.0 (2019-04-17)

LLVM: 6.0.1
Default target: x86_64-pc-linux-gnu
```

Thanks!

**Comment by vladfaust** [comment]
I'm sure it's related to https://github.com/crystal-lang/crystal/issues/7614. @MakeNowJust did your PR fix this case?

**Comment by asterite** [comment]
This is fixed in master, probably fixed by #7614

**Comment by SanksTheYokai** [comment]
Okay, thanks!
